Workbooks/Virtual Machines - Network Dependencies/Connections Overview - Single VM/Connections Overview.workbook

```diff
@@ -491,76 +453,69 @@
             "name": "QueryPadProcessNameTable",
             "type": 1,
             "isRequired": true,
-            "query": "// {Computer:label}\r\nprint(\"let processNamePadding = datatable (ProcessName: string) [];\");",
+            "query": "print(\"let processNamePadding = datatable (ProcessName: string) [];\");",
```

Comment:
why are some of these queries?  am I missing something?  this looks like just a static string that could be part of a different query?  it doesn't have any params, so there's no replacements going on.  am I missing something?

Reply:
Thanks for catching this. Removed.